olive/passes/onnx/conversion.py

```diff
@@ -106,7 +106,8 @@ def _run_for_config(
             for component_name in model.components:
                 # convert each component
                 component_model = model.get_component(component_name)
-                component_output_path = str(Path(output_model_path).with_suffix("") / component_name)
+                # component_name is set with Path(component_model.onnx).stem
+                component_output_path = str(Path(output_model_path).with_suffix("") / f"{component_name}.onnx")
```

Comment:
I think there it might be better to do `{component_name}/model.onnx` so that the external data files between the components don't clash. 
The external data files won't clash if `all_tensors_to_one_file` is `True` but for `False`, both components might have the same names for their tensor data files.

Reply:
So, the change here is not necessary imo

Reply:
Thanks. I did not realized that there will be clash for tensor data. Updated.